src/ml/neural_net/tf_compute_context.cpp

```diff
@@ -155,7 +155,18 @@ float_array_map tf_model_backend::export_weights() const {
   float_array_map result;
   call_pybind_function([&]() {
     // Call export_weights method on the TensorFLowModel
-    pybind11::object exported_weights = model_.attr("export_weights")();
+    pybind11::dict exported_weights = model_.attr("export_weights")();
+
+    // it should be trivial to call this if we use the same interpreter process
+    pybind11::module np = pybind11::module::import("numpy");
+    for (auto& kv : exported_weights) {
+      // defensively call ascontiguousarray to force the
+      // underlying numpy memory layout using the real strides
+      // after this, stride should be vector of 1s
+      exported_weights[kv.first] = np.attr("ascontiguousarray")(
+          pybind11::reinterpret_borrow<pybind11::object>(kv.second));
```

Comment:
refers from here https://github.com/pybind/pybind11/issues/1201

Reply:
for `pybind11::reinterpret_borrow<pybind11::object>`.

Reply:
Per discussion with @hoytak, pybind should handle py::handle to py::object conversion if needed.

Reply:
So I will remove `pybind11::reinterpret_borrow<pybind11::object>`

Reply:
```
In [4]: c = np.ascontiguousarray(b)
In [5]: %timeit  np.ascontiguousarray(c)
790 ns ± 4.12 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```
applying `ascontiguousarray` on array with transformed memory layour is a no-op.